test/fork/adapters/ChainlinkSourceAdapter.sol

```diff
@@ -8,18 +8,8 @@ import {ChainlinkSourceAdapter} from "../../../src/adapters/source-adapters/Chai
 import {DecimalLib} from "../../../src/adapters/lib/DecimalLib.sol";
 import {IAggregatorV3Source} from "../../../src/interfaces/chainlink/IAggregatorV3Source.sol";
 
-contract TestedSourceAdapter is ChainlinkSourceAdapter {
+contract TestedSourceAdapter is ChainlinkSourceAdapter, BaseController {
```

Comment:
I had to make a somewhat opinionated change to this test (as well as all the other fork tests) to get them to work with the `maxAge` under situations where we do multi-round traversal:

due to the previous inheritance structure, we had `maxAge` was set to 0. we have a number of tests that look back over a number of rounds. in conjunction with this 0 value this broke some of the tests. our options to fix this were:
a) implement the `TestedSourceAdapter` to return a value for `maxAge` to something meaningful (like `1 days`) or 
b) add `BaseController` to the inheritance path for the `TestedSourceAdapter` to get the functionality of (a) for free. the upside with this is we also ditch all the `public view override` in all the unit tests.

I opted for (b).

Reply:
Perfect, I agree that's a lot better and much less complicated/copypasta.